packages/compass-crud/src/components/insert-json-document.tsx

```diff
@@ -1,24 +1,48 @@
 import React, { Component } from 'react';
-import PropTypes from 'prop-types';
 
+import { css, cx, palette, withTheme } from '@mongodb-js/compass-components';
 import type { AceEditor } from '@mongodb-js/compass-editor';
 import { Editor, EditorVariant } from '@mongodb-js/compass-editor';
 
-import styles from './insert-json-document.module.less';
+const editorContainerStyles = css({
+  // NOTE: This height is coupled with the min-height of the .editor.
+  padding: '10px 10px 10px 0',
+  height: '300px',
+  overflow: 'auto',
+  flexBasis: 'auto',
+  flexShrink: 1,
+  flexGrow: 1,
+});
+
+const editorContainerStylesLight = css({
+  backgroundColor: palette.gray.light3,
+  borderLeft: `3px solid ${palette.gray.light2}`,
+});
+
+const editorContainerStylesDark = css({
+  backgroundColor: palette.gray.dark3,
+  borderLeft: `3px solid ${palette.gray.dark2}`,
+});
+
+const editorStyles = css({
+  // NOTE: this height is coupled with the padding and height of the .editor-container.
```

Comment:
Looks like this comment and the one on line 8 are now outdated. I'm thinking now because these two are reliant on each other for their height, how do you feel about sharing a variable for the height so there's one source of truth?
Something like `const editorHeight = 280;` and then reference it in both places?